src/metabase/events/notification.clj

```diff
@@ -78,14 +79,19 @@
     {:user-invited-today         (t/format "MMM'&nbsp;'dd,'&nbsp;'yyyy" (t/zoned-date-time))
      :user-invited-email-subject (trs "You''re invited to join {0}''s {1}" (public-settings/site-name) (messages/app-name-trs))
      :user-invited-join-url      (join-url (:object event-info))}
-    {}))
+
+    :event/alert-create
+    {:alert-create-condition-description (->> event-info :object
```

Comment:
ehhh I understand why the name, but don't like it, will look weird in templates (already does)...

Reply:
yea, it's kind of my intention to make it extra explicit :D, to let people it's wonkiness that we did it this way you know.

anw, what's weird about it?